homeassistant/components/homematicip_cloud.py

```diff
@@ -2,13 +2,15 @@
 Support for HomematicIP components.
 
 For more details about this component, please refer to the documentation at
-https://home-assistant.io/components/homematicip/
+https://home-assistant.io/components/homematicip_cloud/
 """
 
+import asyncio
 import logging
-from socket import timeout
 
+from socket import timeout
```

Comment:
`socket` is part of standard library in python.

Reply:
But seems I need to import socket in order to used it for "except timeout:" later.

Reply:
Yes. I'm just saying that the blank line should go between `import socket` and `import voluptuous as vol`.

Reply:
OK, got it.